doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -1920,3 +1920,110 @@ Example of IKE logging:
       ]
     },
   }
+
+Event type: Modbus
+------------------
+
+Common fields
+~~~~~~~~~~~~~
+
+* "id": The unique transaction number given by Suricata
+
+Request/Response fields
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* "transaction_id": The transaction id found in the packet
+* "protocol_id": The modbus version
+* "unit_id": ID of the remote server to interact with
+* "function_raw": Raw value of the function code byte
+* "function_code": Associated name of the raw function value
+* "access_type": Type of access requested by the function
+* "category": The function code's category
+* "error_flags": Errors found in the data while parsing
+
+Exception fields
+~~~~~~~~~~~~~~~~
+
+* "raw": Raw value of the exception code byte
+* "code": Associated name of the raw exception value
+
+Diagnostic fields
+~~~~~~~~~~~~~~~~~
+
+* "raw": Raw value of the subfunction code bytes
+* "code": Associated name of the raw subfunction value
+* "data": Bytes following the subfunction code
+
+MEI fields
+~~~~~~~~~~
+
+* "raw": Raw value of the mei function code bytes
+* "code": Associated name of the raw mei function value
+* "data": Bytes following the mei function code
+
+Read Request fields
+~~~~~~~~~~~~~~~~~~~
+
+* "address": Starting address to read from
+* "quantity": Amount to read
+
+Read Response fields
+~~~~~~~~~~~~~~~~~~~~
+
+* "data": Data that was read
+
+Multiple Write Request fields
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* "address": Starting address to write to
+* "quantity": Amount to write
+* "data": Data to write
+
+Mask Write fields
+~~~~~~~~~~~~~~~~~
+
+* "address": Starting address of content modification
+* "and_mask": And mask to modify content with
+* "or_mask": Or mask to modify content with
+
+Other Write fields
+~~~~~~~~~~~~~~~~~~
+
+* "address": Starting address to write to
+* "data": Data to write
```

Comment:
Could this become too verbose ?

Reply:
Do you mean that outputing all the data could be too verbose? There are three options:
- don't output any data, maybe only the length
- leave as is, always output data
- add config option to choose what to do

Reply:
> Do you mean that outputing all the data could be too verbose?

Indeed, that is what I am wondering.
And there are indeed the 3 options.
What is your feeling about the verbosity of these ?

Reply:
Max length of data will be `255` bytes ad determined by the byte count field, so I would say it is fine for now but an improvement (i.e. new ticket) would be to make this configurable?

Reply:
So, ok for now, and no need to create a new ticket unless the need arises.
(my question is not about the size of one single entry, but also about how often these entries may pop up)

Reply:
Some real world applications have a very small ratio of large entries. Sample pcaps found on the web may have 50% large entries.